src/components/ToDoContainer/index.js

```diff
@@ -1,9 +1,143 @@
 import React, { Component } from "react";
 
+
+let id = 0;
+
+class Add extends Component {
+  constructor(props) {
+    super(props); 
+    this.state = {value : ''};
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({value: event.target.value});
+  }
+
+  handleSubmit(event) {
+    this.props.addTodo(this.state.value);
+    event.preventDefault();
+  }
+  
+  render () {
+    return (
+      <form onSubmit = {this.handleSubmit}>
```

Comment:
Generally, we avoid using form tags - you can just implement this with an `input` tag and a `button` tag (to which you can pass `handleSubmit` as a prop). You've already implemented the state functionality so this shouldn't be that much of a fix.